wrapper/src/combo.c

```diff
@@ -242,7 +243,6 @@ void* combo_threadfunc(void* vmaf_thread_data)
         {
 #ifdef MULTI_THREADING
             thread_data->stop_threads = 1;
-            pthread_mutex_unlock(&thread_data->mutex_readframe);
```

Comment:
Thank you for this! I spotted the same issue and is doing the debugging but you beat me! The weird part is that this issue didn't emerge in Ubuntu when the compiler was g++.